pkg/ccl/changefeedccl/kcjsonschema/kafka_connect_json_schema.go

```diff
@@ -55,63 +59,38 @@ const (
 // spec, but see the source code of org.apache.kafka.connect.data.ConnectSchema
 // for reference.
 type Schema struct {
-	TypeName schemaType
+	TypeName schemaType `json:"type"`
 	// Name is my schema name, optional. Can represent a logical type (such as
 	// decimal) or the overall entity type, such as "envelope", etc.
-	Name schemaName
+	Name schemaName `json:"name,omitempty"`
 	// Field is the name of the field that I am in my parent struct.
-	Field string
+	Field string `json:"field,omitempty"`
```

Comment:
Thoughts on naming this `ParentFieldName` to distinguish it more from `Fields` and to make it clear it's a name?